app.py

```diff
@@ -169,7 +169,25 @@ def dashboard():
         return render_template('dashboard.html', Equipments = users)
     flash("No Equipments available.", "danger")
     return render_template('home.html')
-    
+
+#flask route to view user profile with user history
+@app.route('/profile/<string:username>')
```

Comment:
Using username as the variable will make the URLs unnecessarily long. I think UserId is better to use.